front/src/pages/SearchPage.tsx

```diff
@@ -87,7 +88,14 @@ const SearchPage = ({ saveSearchQuery, savedQueries }: Props) => {
   };
 
   return (
-    <Box mb="25px">
+    <Box 
+      mb="25px"
+      background={colorMode == 'light' ? '#FAFAFA' : '#252e41'}
```

Comment:
Could you alphabetize the props here and change the double equals to a triple equals? Just preference :)  I think then we can merge. thank you!

Reply:
Ok!